Makefile

```diff
@@ -4,11 +4,11 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 NAMESPACE ?= nmstate
 
-NMSTATE_CURRENT_COPR_REPO=nmstate-0.3
-NM_CURRENT_COPR_REPO=NetworkManager-1.26
+NMSTATE_CURRENT_COPR_REPO=nmstate-1.0
+NM_CURRENT_COPR_REPO=NetworkManager-1.30
 
 NMSTATE_FUTURE_COPR_REPO=nmstate
-NM_FUTURE_COPR_REPO=NetworkManager-1.26
+NM_FUTURE_COPR_REPO=NetworkManager-1.30
```

Comment:
We can make this useful by using  nmstate/nmstate-git. nmstate/nmstate is the same as 1.0

Reply:
Done